spec/rubocop/config_loader_spec.rb

```diff
@@ -386,6 +387,14 @@
       )
     end
 
+    it 'loads configuration properly when it includes non-ascii characters ' do
+      create_file(configuration_path, ['# All these cops of mine are ❤',
```

Comment:
No problem. No, it passes for me. I googled the error and  `1.9.3` and OSX appeared. Running any of those?